src/CSET/operators/read.py

```diff
@@ -141,8 +140,9 @@ def read_cubes(
         callback = _create_callback(is_ensemble=True)
         cubes = iris.load(input_files, constraint, callback=callback)
 
-    # Merge cubes now metadata has been fixed.
-    cubes.merge()
+    # Merge and concatenate cubes now metadata has been fixed.
+    cubes = cubes.merge()
+    cubes = cubes.concatenate()
```

Comment:
1) Is it clear why we need or wish to both merge and concatenate input CubeLists? Why has the concatenate command needed to be added here?

2) Given the vagaries of merge and concatenate, particularly for example when likely to have inputs with multiple instances of same variable/STASH, but different attributes (e.g. time processing), do either of these calls require explicit try/fail, or will the code fail in sensible ways if cubes.concatenate() creates it's own error message here?